thirdparty/README.md

```diff
@@ -665,6 +665,17 @@ They can be reapplied using the patches included in the `vhacd`
 folder.
 
 
+## volk
+
+- Upstream: https://github.com/zeux/volk
+- Version: 1.2.170.0 (b18f3192886981f28d17f8f41c26c5a53ac3b977, 2021)
+- License: MIT
+
+Files extracted from upstream source:
+
+- `volk.h`, `volk.c`
+- `LICENSE.md`
+
```

Comment:
```suggestion
- `volk.h`, `volk.c`
- `LICENSE.md`


```
I keep two lines between sections (debatable style choice but it's good to stay consistent :P).

Reply:
Fixed